pkgs/applications/graphics/zgv/switch.patch

```diff
@@ -0,0 +1,14 @@
+GCC complains
```

Comment:
The default case can no longer be empty, and it throws `label at end of compound statement`, and build fails. Zgv hasn't had any updates in a while. I assume that they have not tried compiling with later gcc.